public/data/RajPakhurde.json

```
@@ -0,0 +1,33 @@
+{
+    "name": "Raj Pakhurde",
+    "type": "personal",
+    "bio": "Open Source Enthusiast!",
+    "avatar": "https://github.com/RajPakhurde.png",
+    "links": [
+      {
+        "name": "Follow me on GitHub",
+        "url": "https://github.com/RajPakhurde",
+        "icon": "github"
+      },
+      {
+        "name": "Follow me on Twitter",
+        "url": "https://twitter.com/raj_pakhurde",
+        "icon": "twitter"
+      }
+      {
```

```suggestion
      },
      {
```
Need a comma between array entries.

---

Now i have added comma and removed empty line at end